man/en/sos-report.1

```diff
@@ -362,6 +362,20 @@ be used provided all other required values (case number, upload user) are provid
 Specify a directory to upload to, if one is not specified by a vendor default location
 or if your destination server does not allow writes to '/'.
 .TP
+.B \--upload-method METHOD
+Specify the HTTP method to use for uploading to the provided --upload-url. Valid
+values are 'auto' (default), 'put', or 'post'. The use of 'auto' will default to
+the method required by the policy-default upload location, if one exists.
+
+This option has no effect on upload methods other than HTTPS.
+.TP
+.B \--no-ssl-verify
```

Comment:
`--no-ssl-verify` sounds too generic (is sos verifying some certificates it *collect*?). Maybe `--upload-no-ssl-verify`?

Reply:
Ack.